schemas/test/en/test_thank_you_census_communal_establishment.json

```diff
@@ -7,7 +7,7 @@
     "form_type": "C",
```

Comment:
Why have the census schemas been kept?. We shouldn't have any census specific schemas anymore.